app/main.ipynb

```diff
@@ -496,23 +1705,36 @@
     "lst = [1, 2, 3, 4, 5]"
    ],
    "metadata": {
-    "collapsed": false
-   }
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-11-06T20:06:33.803036Z",
+     "start_time": "2024-11-06T20:06:33.796766Z"
+    }
+   },
+   "outputs": [],
+   "execution_count": 44
   },
   {
    "cell_type": "code",
-   "execution_count": 28,
-   "outputs": [],
-   "source": [
-    "# write your numpy code here and estimate the time of execution"
-   ],
+   "source": "%time result = unknown_signature(lst)",
```

Comment:
The function `unknown_signature` correctly calculates the average of a list using a loop.